.buildkite/pipeline.yml

```diff
@@ -246,8 +246,7 @@ steps:
           build.message !~ /\[only/ && !build.pull_request.draft &&
             build.message !~ /\[skip tests\]/ &&
             build.message !~ /\[skip downstream\]/
-        timeout_in_minutes: 30
-        soft_fail: true
```

Comment:
This is wrong: https://github.com/JuliaGPU/CUDA.jl/commit/6bdb146a755cab9d5f5fdbe703faedabf794ed37

Reply:
Yeah okay will fix, but also I’m now confused from a high level. There an obvious undef var usage in the patch (which I will fix), which breaks CI but I never hit locally just running test.

is this not run as part of the default tests?